specifyweb/frontend/js_src/lib/components/Toolbar/WbsDialog.tsx

```diff
@@ -129,159 +133,246 @@ function TableHeader({
-                      href={`/specify/workbench/${dataset.id}/`}
-                      onClick={
-                        handleDataSetSelect
-                          ? (event): void => {
-                              event.preventDefault();
-                              handleDataSetSelect(dataset.id);
-                            }
-                          : undefined
-                      }
-                    >
-                      <TableIcon
-                        label
-                        name={dataset.uploadplan?.baseTableName ?? 'Workbench'}
-                      />
-                      {dataset.name}
-                    </Link.Default>
-                  </td>
-                  <td>
-                    <DateElement date={dataset.timestampcreated} />
-                  </td>
-                  <td>
-                    <DateElement
-                      date={
-                        dataset.uploadresult?.success === true
-                          ? dataset.uploadresult?.timestamp
-                          : undefined
-                      }
-                    />
-                  </td>
-                  <td>
-                    {canImport && (
-                      <Link.Icon
-                        aria-label={commonText.edit()}
-                        className={className.dataEntryEdit}
-                        href={`/specify/overlay/workbench/${dataset.id}/meta/`}
-                        icon="pencil"
-                        title={commonText.edit()}
-                      />
-                    )}
-                  </td>
-                </tr>
-              ))}
-            </tbody>
-          </table>
-        </nav>
       )}
-    </Dialog>
-  ) : null;
+    </>
+  }
+  className={{
+    container: dialogClassNames.wideContainer,
+  }}
+  dimensionsKey="DataSetsDialog"
+  header={uiSpec.datasetsDialog.header(datasets.length)}
+  icon={icons.table}
+  onClose={handleClose}
+>
+  {datasets.length === 0 ? (
+    <p>
+      {uiSpec.datasetsDialog.empty()}
+    </p>
+  ) : (
+    <nav>
+      <table className="grid-table grid-cols-[1fr_auto_auto_auto] gap-2">
+        <TableHeader sortConfig={sortConfig} onSort={handleSort} />
+        <tbody>
+          {datasets.map((dataset, index) => (
+            <tr key={index}>
+              <td className="min-w-[theme(spacing.40)] overflow-x-auto">
+                <Link.Default
+                  className="font-bold"
+                  href={route(dataset.id)}
+                  onClick={
+                    handleDataSetSelect
+                      ? (event): void => {
+                          event.preventDefault();
+                          handleDataSetSelect(dataset.id);
+                        }
+                      : undefined
+                  }
+                >
+                  <TableIcon
+                    label
+                    name={dataset.uploadplan?.baseTableName ?? 'Workbench'}
```

Comment:
instead of hardcoding `Workbench` here, could you use something like `tables.Workbench.label` instead?